frontend-cert/css-projects/progress-bar/user-stories.md

```diff
@@ -0,0 +1,15 @@
+1. You should have a `div` with an `h2` heading of `Car Acceleration`.
```

Comment:
We should consider allowing campers to have either a `main` or `generic `div` element since either would be correct.
Also, I think we should use an h1 here since this the main title instead of a subheading

```suggestion
1. You should have a `div` or `main` element with an `h1` heading with the text of `Car Acceleration`.
```

Reply:
Also, we should mention, it should have the class of `progress-bar-container` so it will be easier to test that way if they use `div` or `main`